public/video-ui/src/components/VideoAssets/VideoAssets.js

```diff
@@ -110,14 +111,28 @@ class VideoAssets extends React.Component {
     return false;
   }
 
+  renderHeader() {
+    const atomId = this.props.video.id;
+
+    function goToUploadPage() {
+      browserHistory.push(`/videos/${atomId}/upload`);
```

Comment:
Any reason you're using this rather than `<Link />` from react-router... also I wouldn't define a function within the renderHeader function